CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+# 2.14.0 (2018-11-29)
+- Added `stache-hide-from-search` component to allow users to remove content from Stache Search results. [#557](https://github.com/blackbaud/stache2/pull/557)
+- Fixed alignment issue with sidebar and table of contents alignment. [#556](https://github.com/blackbaud/stache2/pull/556)
+- Updated version of `skyux-builder`.
+- Updated Travis build options to enhance test utility.
+- Updated skyux dependencies to depend on components directly. [#553](https://github.com/blackbaud/stache2/pull/553)
+- Updated to allow `Action Button` comopnent to be used without icons.
```

Comment:
Should there be a link to a PR here? I assume we should combine this bullet with the next bullet.

The placement of "to allow" is awkward and hard to follow. "Action Button" should not be capitalized; it should be the element name, which I think should be "sky-action-button" here. Also, "component" is misspelled. ... I'd combine this item with the next one as:

"Updated the `sky-action-button` component to improve accessibility and allow action buttons without icons."